build/scripts/entrypoint-volume.sh

```
@@ -185,41 +185,21 @@ find_projects() {
+configure_no_project() {
+  echo "[INFO] No project found. Using no project mode."
+  PROJECT_PATH=
+}
 
-  if [ "$project_count" -eq 0 ]; then
-    echo "[WARNING] No projects found in $projects_root"
-    PROJECT_PATH="$projects_root"
-    return
-  fi
+configure_single_project() {
+  local project_path="$1"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)